final_prep/annotations.rb

```diff
@@ -16,28 +20,38 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  # shows you the details of your built bear at stored in the Hash
   return built_bear
 end
-
+# Creates 2 instances of the build_a_bear function/method.
```

Comment:
If `build_a_bear` were a class, we'd use the term `instance`; since it's a plain-old-Ruby-method, instead we'd say that line 27 "calls the `build_a_bear` method"